releases/cloud/point-2025-06-17-release-note.md

```diff
@@ -0,0 +1,29 @@
+---
+description: This article lists improvements and bug fixes in the Syskit Point Cloud version 2025.3.96.11
+---
+
+# June 17, 2025
+
+[Start a free trial](https://www.syskit.com/products/point/free-trial/) and [tell us what you think](https://www.syskit.com/company/contact-us/).
+
+## About Syskit Point Cloud
+
+* **New to Syskit Point Cloud?** Explore Syskit Point Cloud with a 21-day [free trial](https://www.syskit.com/products/point/free-trial/) for an easy and effective way to manage and secure your environment.
+
+* **Already using Syskit Point Cloud?** Syskit Point Cloud is automatically upgraded to the latest version when available. The automatic update occurs outside working hours to ensure minimal interference with your day-to-day business. The new version will begin rolling out with this announcement and is expected to reach all customers within the next few days.
+
+## Improvements & Bug Fixes 
+
+* **Improvements made to policies**: Maximum Number of Owners and Minimum Number of Owners.
+  * The policies now **also apply to SharePoint Sites**, starting from the next AutoDiscover sync.
```

Comment:
the default policies apply to SP sites after AD sync
user defined policies can also be applied to SP sites if the user wants that (be it via rules or manually)